modules/samples/oauth2/playground2/src/main/webapp/oauth2.jsp

```diff
@@ -306,6 +321,70 @@
             });
         }
 
+        function sendImplicitFlowIdTokenToServer(idtoken){
+            var data = {
+                "idToken" : idtoken
+            };
+
+            $.ajax({
+                type: 'POST',
+                url: '<%= request.getContextPath()%>/IDTokenExtractorServlet',
+                dataType: 'text',
+                data: data,
+                success: function(data){
+                    if(data != null || data != ""){
+                        var dataJSON = JSON.parse(data);
+
+                        if(dataJSON.hasOwnProperty("user")){
+                            //var loggedInUser = JSON.stringify(dataJSON.user);
```

Comment:
remove commented lines

Reply:
Fixed in commit: 9e2f926d3f750383ef0c41d51056ea1ee401e5cf